samples/Sentry.Samples.GenericHost/Sentry.Samples.GenericHost.csproj

```diff
@@ -21,6 +21,7 @@
   <ItemGroup>
     <ProjectReference Include="..\..\src\Sentry.Extensions.Logging\Sentry.Extensions.Logging.csproj" />
     <PackageReference Include="Microsoft.Extensions.Hosting" Version="8.0.0" />
+    <!-- https://github.com/advisories/GHSA-hh2w-p6rv-4g7w -->
+    <PackageReference Include="System.Text.Json" Version="8.0.4" />
```

Comment:
This is in the sample. Should we bump in the main SDK since it's a major? Could be time.

could cause diamond dependency with apps running .NET 6 that are pinned on S.T.J 6.0.x though so not sure

Reply:
I think maybe we leave it in the samples but roll back similar fixes that I made to SDK... that way we support as far back as possible and pick up on any issues people (who most likely are bumping to address these advisories) might have as a result of conflicts between their own apps and our SDK.